pkg/controller/master/schedulevmbackup/svmbackup.go

```diff
@@ -0,0 +1,122 @@
+package schedulevmbackup
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	harvesterv1 "github.com/harvester/harvester/pkg/apis/harvesterhci.io/v1beta1"
+	"github.com/harvester/harvester/pkg/ref"
+	"github.com/harvester/harvester/pkg/util"
+	utilCatalog "github.com/harvester/harvester/pkg/util/catalog"
+)
+
+const (
+	releaseAppHarvesterName = "harvester"
+)
+
+func deleteCronJob(h *svmbackupHandler, svmbackup *harvesterv1.ScheduleVMBackup) error {
+	cronJob, err := getCronJob(h, svmbackup)
+	if err != nil {
+		return err
```

Comment:
We can check whether cronjob is not found, so we can avoid a ScheduleVMBackup can't be removed.

Reply:
Updated per suggestion, thanks.